mnist-cnn/label_errors_mnist_train_cnn.ipynb

```diff
@@ -4,7 +4,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This example demonstrates the use of `cleanlab`'s experimental module `cleanlab.experimental.mnist_pytorch` to train a CNN model and find label issues in MNIST\n",
+    "# Finding Label Errors in MNIST Dataset with CNNs"
```

Comment:
may be clearer to say:  MNIST Dataset -> MNIST Image Dataset

I think trying to mention the data modality and model type will be good improvements to the descriptiveness of these examples.